### ISSUE TITLE
Specifying <packagename>.<receiver>.<function> fails while line number works

### ISSUE THREAD

**Issue by jmcauley** [derekparker/delve#296]
I am able to specify a break point via filename:linenumber just fine. When I do so, the break command returns the package.receiver.function specification that should correspond to that line number. However, when I try to do: `break package.receiver.function` using the values returned by the first break call, it is not able to find it.
```
(dlv) break github.com/treetopllc/gonoble/graph/submission/dispatcher_service.go:183
Breakpoint 2 set at 0xb2c8f0 for github.com/treetopllc/gonoble/graph/submission.DispatcherService.update ./graph/submission/dispatcher_service.go:183
(dlv) break submission.DispatcherService.update
Command failed: Location "submission.DispatcherService.update" not found
```

My guess is it is because the receiver.function spec (DispatcherService.update in this case) exists in multiple packages.

**Comment by aarzilli** [comment]
It's because it doesn't support partial package names, and the full package name is: `github.com/treetopllc/gonoble/graph/submission`.

**Comment by aarzilli** [comment]
Closed by b5cf1572f812031801f5d77940aa149c8c840c6a
